src/main/java/by/georgprog/epicmusicstore/service/album/AlbumService.java

```diff
@@ -0,0 +1,25 @@
+package by.georgprog.epicmusicstore.service.album;
+
+import by.georgprog.epicmusicstore.dto.AlbumDto;
+import by.georgprog.epicmusicstore.exeption.forbidden.ObtainingDataException;
+import by.georgprog.epicmusicstore.exeption.unauthorized.AlbumNotFoundException;
+import by.georgprog.epicmusicstore.exeption.unauthorized.UserNotFoundException;
+import by.georgprog.epicmusicstore.model.AlbumEntity;
+import by.georgprog.epicmusicstore.service.DaoService;
+
+import java.util.List;
+
+public interface AlbumService extends DaoService<AlbumEntity> {
```

Comment:
I think we don't to extend DaoService of Entity here, because service must not return entity object. We may either remove it or make it of type `DaoService<AlbumDto>`. But I would prefer the first option